src/pages/LandingPage.jsx

```diff
@@ -1,9 +1,49 @@
 import React from 'react';
+import '../stylesheets/LandingPage.scss';
+import Logo from '../assets/Light_Logo.png';
+import LogoText from '../assets/Light_Logo_Text.png';
+import HYLButton from '../components/HYLButton.jsx';
+
 
 const LandingPage = () => (
-  <div>
-    Landing Page
-  </div>
+  <>
+    {/* Replace this div with actual navbar */}
+    <div className="tempNavBar">
+        This is where Nav Bar will go!
+    </div>
+
+    <div className="mainBody">
+      <img src={LogoText} className ="logoText" />
+      <div className="eventDetails">
+        <div className="registerButtons">
+          <HYLButton text="Become a Hacker" color="blue" />
+          <HYLButton text="Become a Mentor" color="red"/>
+        </div>
+        <a className="date">March 27, 2021 • Virtual event</a>
```

Comment:
This shouldn't be in an `<a>` tag, the content has no reason to link to something else